website/docs/guides/markdown-features/markdown-features-admonitions.mdx

```diff
@@ -69,6 +69,27 @@ Some **content** with _markdown_ `syntax`. Check [this `api`](#).
 
 :::
 
+## Usage with Prettier
```

Comment:
I'd have used an admonition to relay this message but there are already so many example admonitions on this page people probably won't notice a non-example

Reply:
That's hard🤦‍♂️Maybe we should wrap the demo admonitions in `<BrowserWindow>` instead? But the current solution looks fine